### PR TITLE
WFDR-115-Add_Prefs_Model

### PR DESCRIPTION
**Pull request by NickMoo5**
## Summary of changes
Added user preferences to model

## Trello Task link
https://trello.com/c/kvjvXj50/24-115-add-preferences-to-user-model

## Corresponding User Story Number and Title
User Story: 1 - Making a new Account

## Detailed list of changes
Added attributes to the user model
-accessibility
-dietary restriction
-language(1-3)

Added corresponding getter and setters

### CODE REVIEW

File: src/main/java/sfu/cmpt276/project/model/User.java [hunk 1]
```diff
@@ -13,20 +13,70 @@ public class User {
...
-    public User(String username, String password, String firstName, String lastName, String email) {
+    public User(String username, String password, String firstName, String lastName, String email, String accessibility, String dietaryRestriction, 
+                String languageOne, String languageTwo, String languageThree) {
         this.username = username;
         this.password = password;
         this.firstName = firstName;
         this.lastName = lastName;
         this.email = email;
         this.accountType = ACCOUNTTYPE_USER;
+        this.accessibility = accessibility;
```
Comment (NickMoo5) [inline src/main/java/sfu/cmpt276/project/model/User.java:42]: Test Comment

**Review by NickMoo5 — COMMENTED** [review]
Last Comments

**Review by danielqyu — APPROVED** [review]
looks good to me. Will need to update editPrefs page to include these specific names

**Review by setaDM — APPROVED** [review]
Code looks great. Should be easy to do the backend for the preferences on signup now.

**Pull request merged**